lib/bambora/bank/adapters/payment_profile_response.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module Bambora
+  module Bank
```

Comment:
This adapter is very specific to the bank profile endpoint, so I wanted to keep it in the bank name space.